upf/upf_cli.c

```diff
@@ -1197,7 +1221,7 @@ VLIB_CLI_COMMAND (upf_show_session_command, static) =
 {
   .path = "show upf session",
   .short_help =
-  "show upf session",
+  "show upf session [up seid 0x...] [0x... flows] [limit N]",
```

Comment:
do not mix prefix and postfix parameters. VPP cli command always use the prefix form, stick with that please.

The VPP CLI is not operation tool! The usability of the command is not really important. But, you create a potential of messing up the command parsing if you mix postfix and prefix arguments like that.

Reply:
Also, limiting the output is pretty useless. Either you know that it is safe to list all or there are too many session that listing even a part of them is useless anyhow.

I would suggest to change the command to either display a single session or all. Use a special flag to force all, e.g.:
```
"show upf session [ up seid N [flows N] | all]"
```

Reply:
If we have like 1k sessions and want to limit the output, so running like `show upf session limit 100` will always show us first 100 sessions, and the rest 900 will not be accessible - either there should be paging (like `limit 100-200`) or just remove the limit... 

But maybe if first entries can say us something meaningful than it can stay, what we can gain by seeing first sessions?

Reply:
When I reread the commit description the help should be:
```
show upf session [up seid 0x...][up seid 0x... flows][limit N]
```
Right?